### DISCUSSION TITLE
Translate to Ukrainian

### DISCUSSION THREAD

**Discussion by artiekra** [mbrlabs/Lorien#258] [General]
Seems like there is no ukr translation yet, and I wish to add the translations to the project.
Wondered if there any work done on it so far? Also, never worked with Godot before, and I suppose I would have to build the project to test out the translations. Are there any problems that can occur with them, that would require me to test them locally first?

**Comment by mbrlabs** [comment]
Cool! No, no one did a ukrainian translation yet. You can have a look at the localization guide here: https://github.com/mbrlabs/Lorien/blob/main/docs/i18n.md

As for testing it out, you just have to open the project in Godot 3.5.x i.e. the LTS version (not 4; i'm in the process of porting it but the master branch is still on version 3.x) and start the project (no code changes required; it will just pick up the language automatically when it's placed into the right directory).

If you don't want or can't run it locally, there is also a CI process that runs on every pull request, which builds the project automatically. You can download the artifacts and check it out this way as well.
